### PR TITLE
add weekly changelog 2.272~2.275 & lts changelog 2.263.2

### PR DESCRIPTION
**Pull request by zhaoying818**
### 检查项（Submitter checklist）

- [ ] 已添加 Issue
- [x] 已阅读[翻译规范](https://github.com/jenkinsci/localization-zh-cn-plugin/blob/master/specification.md)
- [x] [原文地址](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)

### Desired reviewers

@jenkins-infra/chinese-localization-sig

### CODE REVIEW

File: content/_data/changelogs/lts.yml [hunk 1]
```diff
@@ -3478,6 +3478,78 @@
...
+      - jvz
+      - Wadeck
+  - type: major bug
+    category: regression
+    pull: 5099
+    issue: 64373
+    authors:
+    - timja
+    message: |-
+      帮助文本现在扩展了GitHub组织文件夹的行为(由 2.244 引入的缺陷回归)。
```
Comment (yJunS) [inline content/_data/changelogs/lts.yml:3501]: `GitHub` 两边没有加空格

File: content/doc/upgrade-guide/2.263.adoc [hunk 2]
```diff
@@ -8,6 +8,123 @@ notitle: true
...
+
+[#formvalidation]
+===== 表单验证响应的安全性强化
+// /* package */ static /* non-final for Groovy */ boolean APPLY_CONTENT_SECURITY_POLICY_HEADERS = SystemProperties.getBoolean(FormValidation.class.getName() + ".applyContentSecurityPolicyHeaders", true); 
+
+现在，使用标准 Jenkins API 实现的表单验证会在响应上设置限制性的 Content-Security-Policy 标头。
+
+这将防止表单验证响应中的跨站点脚本漏洞被利用为反射型 XSS。
+
+NOTE: 者并_不_意味着这些漏洞不再是问题，利用将变得更加困难，例如，它们可能需要额外的权限才能设置。
```
Comment (yJunS) [inline content/doc/upgrade-guide/2.263.adoc:98]: `者` -> `这`

**Review by yJunS — CHANGES_REQUESTED** [review]

File: content/doc/upgrade-guide/2.263.adoc (hunk 2 above)
Comment (zhaoying818) [reply content/doc/upgrade-guide/2.263.adoc:98]:
```suggestion
NOTE: 这并_不_意味着这些漏洞不再是问题，利用将变得更加困难，例如，它们可能需要额外的权限才能设置。
```

File: content/_data/changelogs/lts.yml (hunk 1 above)
Comment (zhaoying818) [reply content/_data/changelogs/lts.yml:3501]:
```suggestion
      帮助文本现在扩展了 GitHub 组织文件夹的行为(由 2.244 引入的缺陷回归)。
```

**Review by yJunS — APPROVED** [review]

**Pull request merged**